script.js

```diff
@@ -1,11 +1,16 @@
+const TODOS_SESSION_KEY = "todos";
+
+const storedTodos = JSON.parse(sessionStorage.getItem(TODOS_SESSION_KEY));
+const initialState = {
+  todos: storedTodos || [],
+  filter: "all",
+};
+
 const form = document.getElementById("todoForm");
 const input = document.getElementById("inputField");
 const list = document.getElementById("list");
 
-const state = {
-  todos: [],
-  filter: "all",
-};
+const state = { ...initialState };
```

Comment:
There's no point of `initialState`. Assigning state object directly to `state` would do absolutely the same.